Core/Services/StitchCoreRemoteMongoDBService/Tests/StitchCoreRemoteMongoDBServiceTests/CoreRemoteMongoCollectionUnitTests.swift

```diff
@@ -1142,15 +1142,15 @@ final class CoreRemoteMongoCollectionUnitTests: XCMongoMobileTestCase {
         exp.isInverted = true
         try expectStreamToClose(expectation: exp)
         coreRemoteMongoClient.onRebindEvent(AuthRebindEvent.userLoggedIn(loggedInUser: testUser))
-        wait(for: [exp], timeout: 0.1)
+        wait(for: [exp], timeout: 10)
```

Comment:
[q] the expectation is inverted, so shouldn't these be testing that the expectation never gets fulfilled? was this actually failing before? why did this need to change?

Reply:
This test was flaky before. I assumed it was related to timeout.